test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+index: 0
+name: "root"
+branch_node {
+  branches {
+    condition {
+      name: "cookie vs. cookieless. name field is optional."
```

Comment:
I saw the examples have a name in this format: "<name>.<name>.<name>". For the purposed of parsing this proto, this can be anything right? Also, I will change the op to TRUE and remove the value field

Reply:
Which example? Maybe it is the name of a model node? This is a FieldFilterProto.
When loading the model proto it will check that this name string is a valid proto field name.

Reply:
<img width="783" alt="Screen Shot 2021-06-03 at 1 21 58 PM" src="https://user-images.githubusercontent.com/65198074/120686154-ae9e7900-c46e-11eb-858a-a7ec16a60107.png">

Reply:
I just noticed that github markdown changed what I had written. This picture above shows the format that I was mentioning

Reply:
The list of <name> form a path from the top level proto to the target field, as demonstrated in the unit tests.